test/parallel/test-fs-readfile-flags.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+// Test of fs.readFile with different flags.
+const common = require('../common');
+const fs = require('fs');
+const assert = require('assert');
+const path = require('path');
+const tmpdir = require('../common/tmpdir');
+
+tmpdir.refresh();
+
+{
+  const emptyFile = path.join(tmpdir.path, 'empty.txt');
```

Comment:
nit: all files in this test are empty, so `emptyFile` doesn't provide a lot of value. Suggestion: rename to `existingFile` or smth similar.